README.rst

```diff
@@ -6,6 +6,29 @@ X-Ray Timing Made Easy
 
 We are writing a general-purpose timing package for X-ray time series.
 
+Note to Users
+-------------
+
+Stingray is currently in **development phase**. Some of the code is not in
+its final stage, and might change before our first release. There might also
+still be bugs we are working on fixing and features that are not finished.
+
+We encourage you to download it and try it out, but please be aware of
+the caveats of working with in-development code.
+At the same time, we welcome contributions and we need your help!
+If you have your own code duplicating any part of the methods implemented in
+Stingray, please try out Stingray and compare to your own results.
+
+We do welcome any sort of feedback: if something breaks, please report it via
+the `issues<https://github.com/dhuppenkothen/stingray/issues>`_. Similarly,
```

Comment:
I think there needs to be a gap between `issues` and the angular bracket. We can see the "rich diff" and see how it will look. Also maybe `StingraySoftware/singray/issues` ?